examples/textures/textures_image_drawing.c

```diff
@@ -93,4 +93,4 @@ int main(void)
     //--------------------------------------------------------------------------------------
 
     return 0;
-}
```

Comment:
Not sure why this shows up in the diff.

Reply:
It seems the close brake has been deleted, please, could you review it?

Reply:
> It seems the close brake has been deleted, please, could you review it?

the brace was unaffected. github is showing that the file originally did not have an empty line at the end, and now one has been added. im guessing @FluxFlu has some linter program running that did it automatically